lib/hacluster.pm

```diff
@@ -651,7 +661,7 @@ sub ha_export_logs {
 Check state of the cluster. This will call B<$crm_mon_cmd> to check the current
```

Comment:
```suggestion
Check state of the cluster. Calls B<$crm_mon_cmd> and inspect its output checking:
  - the current status of the cluster,
  - for inactive resources,
  - for S<partition with quorum>
 Checks that the reported number of nodes in the output
of C<crm node list> and B<$crm_mon_cmd> is the same.
Runs C<crm_verify -LV>.
With the named argument B<proceed_on_failure> set to 1, the method will use
B<script_run()> and attempt to run all commands in SUT without checking for errors.
```

Reply:
Just more schematic form

Reply:
Done .... although with this more schematic form, I need to drop the bit about the Oxford comma. ;)